projects/go-tester/src/app/app.module.ts

```diff
@@ -40,7 +43,8 @@ import { AppService } from './app.service';
   ],
   entryComponents: [
     GoButtonComponent,
-    GoIconComponent
+    GoIconComponent,
+    GoIconButtonComponent
```

Comment:
This won't affect the implementation of the button at all, but we actually don't need to add the component to the `entryComponents`. We only need to do this if we're injecting the `GoIconButtonComponent` into our modal or off-canvas. In this case, it doesn't seem like we are so we can probably remove the component from this array and actually remove the import for the component. **not the module though**

Reply:
I think this was for testing purposes. It might be helpful for all of us to get together and chat about this. I don't fully understand this within the context of this project yet.

Reply:
I removed the component from the `entryComponents` after our offline discussion.